extensions/vscode/src/extension.ts

```diff
@@ -0,0 +1,67 @@
+import * as vscode from 'vscode';
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+} from 'vscode-languageclient/node';
+
+
+let client: LanguageClient;
+
+export async function activate(context: vscode.ExtensionContext) {
+  let wakePath: string | undefined | null = vscode.workspace.getConfiguration('wakeLanguageServer').get('path');
+
+  if (wakePath === null || wakePath === undefined) {
+    const selection = await vscode.window.showWarningMessage('The path to the wake binary is not set. Highlighting will work but the LSP will not!', 'Configure', 'Dismiss');
+
+    if (selection === 'Configure') {
+      vscode.commands.executeCommand( 'workbench.action.openSettings', 'wakeLanguageServer.path' );
```

Comment:
The parens having spaces before/after them on this line is part of the lint rules?

Reply:
uhh not sure. It passed the lint check but I don't think that includes formatting. The space thing is just something I've picked up from other "modern" programming languages but I don't feel strongly about it

Reply:
I was mostly making sure we ran the lint checks. If the linter lets you both put and not put spaces after the parens thats fine.